assessment/asr_validation/wald2017B/analysis/asr_concrete_block_uniaxial_rebar.i

```diff
@@ -154,7 +154,7 @@
     type = EqualValueEmbeddedConstraint
     secondary = 2
     primary = 1
-    penalty = 31e6
+    penalty = 31e9
```

Comment:
We should change these to a number that is simply something like 1e9 so it doesn't look like it's physically meaningful.  Also, I think these work with even higher numbers, maybe 1e12 (at least that's been my experience on similar models).

Reply:
I was having convergence issues with higher numbers. This is as high as I could go.